RELEASE.md

```diff
@@ -0,0 +1,47 @@
+Release type: minor
+
+You can now configure your schemas to provide a custom subclass of
+`strawberry.types.Info` to your types and queries.
+
+```py
+import strawberry
+from strawberry.schema.config import StrawberryConfig
+
+from .models import ProductModel
+
+
+class CustomInfo(strawberry.Info):
+    def is_selected(self, field: str) -> bool:
+        """Check if the field is selected on the top-level of the query."""
+        return field in [sel.name for sel in info.selected_fields]
```

Comment:
I feel like this should in Strawberry :D 

Also I'm worried people will copy this without knowing how it filly works, could we make a different example? 😊

Reply:
I have considered it but the implementation we currently have (which is recursive) needs refactoring. I have since got a slightly different utility for a slightly different use-case which would probably be better here anyways, so I'll use that.

If I ever feel the `is_selected` is in a state to contribute in, I definitely will! 😄 Just a case of figuring out how best to make it work in as many cases as possible (and appropriate).

Reply:
So I've got a completely different example from that (currently only in local, will push it all once I've done the docs), hopefully it's a better example. All our use cases for this are quite complex so it's tricky to think of something simple 😅